packages/react-day-picker/src/hooks/useDayModifiers/useDayModifiers.ts

```diff
@@ -0,0 +1,51 @@
+import { ModifierStatus } from '../../types/Modifiers';
+import { useDayPicker } from '../../contexts/DayPicker';
+import { useModifiers } from '../../contexts/Modifiers';
+import { getModifierStatus } from './utils/getModifierStatus';
+
+export type DayModifiers = {
+  /** The status of the modifiers */
+  modifiers: ModifierStatus;
```

Comment:
I don't like much a `modifiers` have a  `ModifierStatus` type, when we also have `Modifiers` type:

- rename `ModifierStatus` to `ModifiersStatus`
- evaluate this suggestion (but may be the API comes less elegant?)

```suggestion
  modifiersStatus: ModifiersStatus;
```